torch/csrc/jit/tensorexpr/stmt.h

```diff
@@ -3,6 +3,7 @@
 #include <algorithm>
 #include <list>
 #include <string>
+#include <unordered_set>
```

Comment:
What's going on here?

Reply:
Oh this is because you are no longer recursively including typeid.h? I have always found that I regretted doing header changes in PRs, because there's always more to fix in fbcode later. If you run into more resistance you should probably consider undoing the header change.

Reply:
The change lets the TypeMeta stuff depend directly on ScalarType, but yeah, I could substitute some new transitive inclusion if it jitters fbcode too much.